src/quo2/components/title/style.cljs

```diff
@@ -0,0 +1,18 @@
+(ns quo2.components.title.style
+  (:require
+    [quo2.foundations.colors :as colors]))
+
+(def default-margin 20)
```

Comment:
do we need to add this? Often it is easier to add the margin/padding at the page level and then all components within that page view container can sure that padding.